pkg/kubelet/kubelet_pods.go

```diff
@@ -146,6 +147,11 @@ func makeMounts(pod *v1.Pod, podDir string, container *v1.Container, hostName, h
 
 			hostPath = filepath.Join(hostPath, mount.SubPath)
 
+			err = volumevalidation.ValidatePathNoBacksteps(hostPath)
```

Comment:
Good catch. Since hostPath is checked in the mounter I moved the check to the top of the conditional and only check the SubPath.